piety-promoted-v1/modernized/02-piety-promoted-v1.adoc

```diff
@@ -4088,6 +4099,10 @@ with many more sensible expressions which she uttered.
 That night she received a letter from our friend William Penn, whom she much esteemed,
 and who had been to visit her in the beginning of her sickness; part of it is as follows:
 
+[.embedded-content-document.letter]
+--
+
+[.salutation]
 Dear Elizabeth,
 
 '`I am grieved that I am hindered from seeing you, but the Lord I have sought for you,
```

Comment:
this just needs to have the quotes (apostrophes) removed from before and after the letter (because its embedded).